test/test_helper.rb

```diff
@@ -23,15 +23,23 @@ class Minitest::Test
   create_table "chairs", force: true do |t|
     t.string "name"
   end
+  create_table "cushions", force: true do |t|
+    t.integer "chair_id"
+  end
 end
 
 class Leg < ActiveRecord::Base
-  belongs_to :chair
+  belongs_to :chair, foreign_key: "chair_id"
+end
+
+class Cushion < ActiveRecord::Base
+  belongs_to :chair, foreign_key: 'chair_id'
 end
 
 class Chair < ActiveRecord::Base
   has_many :legs
   has_many :feet, class_name: 'Leg'
+  has_many :curshion
```

Comment:
this has a typo and is not being used

Reply:
Fix the typo [here](https://github.com/holistics/prosopite/pull/1/commits/ac7335ac2de4e4e43efb98fddbcd14e0c2023348). Even though it is not being used, I still decide to keep it for consistency.